### DISCUSSION TITLE
JWT example with Rails

### DISCUSSION THREAD

**Discussion by bdavidxyz** [janko/rodauth-rails#78] [Ideas]
Christmas is soon, so what I would like to add on my wishlist is an example of the JSON API feature with JWT on the rails demo app. Docs stated it is possible, but a simple example would be nice to grab the concept. Thanks for the wonderful work BTW !

**Comment by janko** [comment]
I could modify the Rails demo to add JSON API access with JWT. I initially tried creating a new demo app, which would also contain a SPA in React that consumes that API, but that proved to be too much work. Would it be sufficient to just enable JWT, and show some example API requests in Ruby in the README?

**Reply by bdavidxyz** [reply]
Hi @janko, thanks for the reply. Yes, it would be really sufficient to just enable JWT, with an example of API request, either in the README, or inside a view partial. Maybe both. Anyway this would be a great demo of the strength of rodauth, compared to Devise or Clearance, where adding this feature is really tricky.

**Reply by janko** [reply]
Added in https://github.com/janko/rodauth-demo-rails/commit/79d1b1becbb0551e2f4856039fbae7bcbe03e17e.
